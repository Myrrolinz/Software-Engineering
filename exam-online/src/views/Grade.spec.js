import { shallowMount } from '@vue/test-utils';
import Grade from './Grade';





describe('<Grade/>', () => {
	const wrapper = shallowMount(Grade);

	// 快照测试
	it('snapshot测试', () => {
		const wrapper2 = shallowMount(Grade);
		expect(wrapper2.html()).toMatchSnapshot()
		wrapper2.destroy()
    })

	
  });

  /*
  it("slide correctly", () => {
		// 滑动 slide
		wrapper.find('slide-verification').trigger('check-result');
	  });
  		// 可以立即获取 msg 最新的值，不再需要 wrapper.vm.$nextTick();
		expect(wrapper.find('h1').text())
		  .toEqual('new message');
		  */
