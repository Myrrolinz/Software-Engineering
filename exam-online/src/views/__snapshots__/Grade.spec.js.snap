// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Grade/> snapshot测试 1`] = `
"<div id=\\"grade\\">
  <el-table-stub data=\\"\\" fit=\\"true\\" border=\\"true\\" showheader=\\"true\\" selectonindeterminate=\\"true\\" indent=\\"16\\" treeprops=\\"[object Object]\\" style=\\"width: 100%;\\">
    <el-table-column-stub type=\\"index\\" width=\\"50\\" resizable=\\"true\\" filtermultiple=\\"true\\" sortorders=\\"ascending,descending,\\"></el-table-column-stub>
    <el-table-column-stub type=\\"default\\" label=\\"评卷时间\\" prop=\\"creat_time\\" sortable=\\"true\\" resizable=\\"true\\" filtermultiple=\\"true\\" sortorders=\\"ascending,descending,\\"></el-table-column-stub>
    <el-table-column-stub type=\\"default\\" label=\\"考试名称\\" prop=\\"exam.name\\" width=\\"300\\" resizable=\\"true\\" filtermultiple=\\"true\\" sortorders=\\"ascending,descending,\\"></el-table-column-stub>
    <el-table-column-stub type=\\"default\\" label=\\"得分\\" prop=\\"score\\" width=\\"150\\" sortable=\\"true\\" resizable=\\"true\\" filtermultiple=\\"true\\" sortorders=\\"ascending,descending,\\"></el-table-column-stub>
    <el-table-column-stub type=\\"default\\" label=\\"考试时间\\" prop=\\"exam.exam_date\\" width=\\"180\\" sortable=\\"true\\" resizable=\\"true\\" filtermultiple=\\"true\\" sortorders=\\"ascending,descending,\\"></el-table-column-stub>
    <el-table-column-stub type=\\"default\\" label=\\"筛选\\" prop=\\"score\\" width=\\"200\\" resizable=\\"true\\" filtermethod=\\"[Function]\\" filters=\\"[object Object],[object Object]\\" filterplacement=\\"bottom-end\\" filtermultiple=\\"true\\" sortorders=\\"ascending,descending,\\"></el-table-column-stub>
  </el-table-stub>
  <pagination-stub></pagination-stub>
</div>"
`;
